### PR TITLE
docs: fix quick-start create subscription sink 

### PR DESCRIPTION
**Pull request by xdlbdy**
update sink

### CODE REVIEW

File: user-manual/getting-started/quick-start.md [hunk 1]
```diff
@@ -179,7 +179,7 @@ vanus-display-**************     1/1     Running   0          12s
 ```shell
 vsctl subscription create \
   --eventbus quick-start \
-  --sink 'http://quick-display:80' \
+  --sink 'http://quick-display.vanus:80' \
```
Comment (wenfengwang) [inline user-manual/getting-started/quick-start.md:182]: remove `.vanus` or modify other `--sink`  in order to have the same style

**Review by wenfengwang — CHANGES_REQUESTED** [review]

File: user-manual/getting-started/quick-start.md [hunk 2]
```diff
@@ -195,7 +195,7 @@ output should like below
 +---------------------+-------------+-----------------------------------+----------------------------------------------+-------------+
 |          ID         |   EVENTBUS  |                SINK               |                    FILTER                    | TRANSFORMER |
 +---------------------+-------------+-----------------------------------+----------------------------------------------+-------------+
-| 1656331986533051044 | quick-start | http://vanus-display.default:80   | [                                            | ""          |
+| 1656331986533051044 | quick-start | http://vanus-display.vanus:80   | [                                            | ""          |
```
Comment (wenfengwang) [inline user-manual/getting-started/quick-start.md:198]: align table boxes

**Review by wenfengwang — CHANGES_REQUESTED** [review]

File: user-manual/getting-started/quick-start.md (hunk 1 above)
Comment (xdlbdy) [reply user-manual/getting-started/quick-start.md]: remove `.vanus`

File: user-manual/getting-started/quick-start.md (hunk 2 above)
Comment (xdlbdy) [reply user-manual/getting-started/quick-start.md]: done

File: user-manual/getting-started/quick-start.md [hunk 3]
```diff
@@ -335,4 +335,4 @@ the transformed event has been displayed.
 
 ## Next
 
-You can go forward to [How to use Vanus](../how-to) for learning more about Vans.
\ No newline at end of file
+You can go forward to [How to use Vanus](../how-to) for learning more about Vans.
```
Comment (wenfengwang) [inline user-manual/getting-started/quick-start.md:338]:
```suggestion
You can go forward to [How to use Vanus](../how-to) for learning more about Vanus.
```

**Review by wenfengwang — CHANGES_REQUESTED** [review]

**Review by wenfengwang — APPROVED** [review]

**Pull request merged**